### PR TITLE
feat(config): add support for TypeScript

### PR DESCRIPTION
**Pull request by wesleycho**
- Add TypeScript support for `karma.conf.ts`

This should address #1727.

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -94,10 +94,14 @@ var processArgs = function (argv, options, fs, path) {
       configFile = './karma.conf.js'
     } else if (fs.existsSync('./karma.conf.coffee')) {
       configFile = './karma.conf.coffee'
+    } else if (fs.existsSync('./karma.conf.ts')) {
+      configFile = './karma.conf.ts'
     } else if (fs.existsSync('./.config/karma.conf.js')) {
       configFile = './.config/karma.conf.js'
     } else if (fs.existsSync('./.config/karma.conf.coffee')) {
       configFile = './.config/karma.conf.coffee'
+    } else if (fs.existsSync('./config/karma.conf.ts')) {
```
Comment (maksimr) [inline lib/cli.js]: Here should be `.config`

**Comment by dignifiedquire** [comment]
Would be good to have some test for this.

**Comment by wesleycho** [comment]
Any tip as to where to look for putting a test? I didn't see anything for coffeescript or livescript, and the way they are set up, it may not be so simple to set up since support is checked globally.

**Comment by wesleycho** [comment]
Found it, added a test. I also added a config template and code to the formatters to mirror the coffeescript and livescript code, although it seems they are not used.

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**